index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <meta charset="utf-8">
+  <style>
+  @import url('https://fonts.googleapis.com/css?family=Marck+Script|Sawarabi+Mincho');
+</style>
+<link rel="stylesheet" href="styles/style.css" type="text/css">
+<link rel="stylesheet" href="styles/normalize.css" type="text/css">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Marck Script ||Sawarabi Mincho">
+<title>Index</title>
+</head>
+<p>
+  <a href="Index.html">Index</a>
+  <a href="About.html">About</a>
+  <a href="positivity.html">Positivity</a>
+  <a href="codingschool.html">Coding School</a>
+</p>
+<h1>Hello friends!</h1>
+<h2>Welcome to here</h2>
+
+
+<img src="http://placekitten.com/600/300" alt="placekitten" >
+<p class="flex-container">
+  <p class="item1"> I'm not comfortable with personal websites so here is my best impersonation of a personal website.</p>
+
+
+
+  <div class="footer">
```

Comment:
There is already a footer tag that we went over explicitly, I'm not sure why you need a div here instead.